pyhn/hnapi.py

```diff
@@ -185,9 +185,12 @@ def get_published_time(self, source):
         """
         Gets the published time ago
         """
-        p = re.compile(r'\d{1,} (minutes|minute|hours|hour|day|days) ago')
-        results = p.search(source)
-        return results.group()
+        p = re.compile(r'\d{1,}\s(minutes|minute|hours|hour|day|days)\sago', re.U)
```

Comment:
Good catch, code updated, sorry for this late reply, Now is daytime in Beijing, China. haha. BTW, I created the issue I met, and may have a look at the source code after work.